test/mocha/20-api.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2015-2016 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+var async = require('async');
+var bedrock = require('bedrock');
+var brIssuer = require('bedrock-issuer');
+var eventLog = require('bedrock-event-log').log;
+var helpers = require('./helpers');
+var mockData = require('./mock.data');
+var uuid = require('uuid').v4;
+
+describe('node API', () => {
+  before('Prepare the database', function(done) {
+    helpers.prepareDatabase(mockData, done);
+  });
+  after('Remove test data', function(done) {
+    helpers.removeCollections(done);
+  });
+  describe('issue API', () => {
+    it('should issue a credential with a null actor', done => {
+      bedrock.events.on(
+        `bedrock-issuer.credential.CredentialIssue`, (e, callback) => {
+          eventLog.add(e, callback);
+        });
+      var credential = bedrock.util.clone(mockData.unsignedCredential);
+      credential.issuer = mockData.identities.organizationAlpha.identity.id;
+      credential.id = uuid();
+      async.auto({
+        issue: callback => {
+          brIssuer.issue(null, credential, callback);
+        },
+        findEvent: ['issue', callback => {
+          eventLog.find({}, callback);
+        }],
+        test: ['findEvent', (callback, results) => {
+          var r = results.findEvent;
+          r.should.be.an('array');
```

Comment:
I think modern usage is something like this:
`r.should.be.an.Array()`
and
`r.should.be.an.Object()` below